server/subsonic/api.go

```diff
@@ -71,12 +73,12 @@ func (api *Router) routes() http.Handler {
 
 	// Subsonic endpoints, grouped by controller
 	r.Group(func(r chi.Router) {
-		r.Use(getPlayer(api.players))
+		r.Use(server.RealIPMiddleware, getPlayer(api.players))
```

Comment:
I don't think we need to explicitly apply the RealIPMiddleware to Subsonic endpoints, as it is already added as part of the default middlewares in server.go/initRoutes(), right?

Reply:
No, and that's why you were calling `realIP` in `getPlayer` before. I don't have the routes setup in mind right now, but if you are not satisfied with this answer I can have another look to give a more detailed explanation.

Reply:
Maybe I added `realIP` in `getPlayer` before adding the `realIPMiddleware` in https://github.com/navidrome/navidrome/pull/2858/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R167

We should have a test for this, to be sure it is applied correctly and not double applied.

Reply:
Ah I see now how it is supposed to work, thanks for the pointers.
I think that because of the call to `realIP` inside of `getPlayer` I got confused and thought the route configs were separate.
I'll fix this, might take a few days.

Reply:
Awesome! I have a release ready to go, just waiting for this one. Do you think we will get this done until next weekend? No pressure, I can release a version now, and add this PR to the next bug fix release (in a month or so). Let me know.

Reply:
Yes, I plan to work on it tomorrow. It should be done at the latest by the middle of the week.

Reply:
Done. I didn't add an extra test as the middleware is applied exactly once in code.